### PR TITLE
Google provider delete deprecated reaching removal date (December 2024)

### PR DESCRIPTION
**Pull request by olegkachur-e**
---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by potiuk — APPROVED** [review]

File: providers/src/airflow/providers/google/CHANGELOG.rst [hunk 1]
```diff
@@ -80,6 +103,8 @@ Breaking changes
     * Removed ``CreateHyperparameterTuningJobOperator.sync``. This parameter is not in actual use
     * Removed ``CustomTrainingJobBaseOperator.sync``. This parameter is not in actual use
     * Removed ``GKEStartPodOperator.get_gke_config_file()``. Please use ``GKEStartPodOperator.fetch_cluster_info()`` instead
+    * Removed ``CreateDataPipelineOperator``. Please use the ``DataflowCreatePipelineOperator`` instead
+    * Removed ``RunDataPipelineOperator``. Please use the ``DataflowRunPipelineOperator`` instead
```
Comment (eladkal) [inline providers/src/airflow/providers/google/CHANGELOG.rst:107]: This is not right
You can't add remove stuff to old versions. The code existed in version 11.0

Comment (potiuk) [reply providers/src/airflow/providers/google/CHANGELOG.rst:107]: ah ..  missed it

Comment (olegkachur-e) [reply providers/src/airflow/providers/google/CHANGELOG.rst:107]: Oh, my fault. I'll introduce the PR with a fix.

Comment (olegkachur-e) [reply providers/src/airflow/providers/google/CHANGELOG.rst:107]: https://github.com/apache/airflow/pull/45127

Comment (eladkal) [reply providers/src/airflow/providers/google/CHANGELOG.rst:107]: I already fixed it during release prepration

**Pull request merged**